stdlib/range.gr

```diff
@@ -0,0 +1,25 @@
+export enum Range {
+  Inclusive(Number, Number),
+  Exclusive(Number, Number),
+}
+
+export let isBetween = (value, range) => {
+  match (range) {
+    Inclusive(lower, upper) when value >= lower && value <= upper => true,
+    Exclusive(lower, upper) when value >= lower && value < upper => true,
+    _ => false
+  }
+}
```

Comment:
This looks like it doesn't handle the cases where `Inclusive(upper, lower)` and `Exclusive(upper, lower)`.

Reply:
Done!